lighthouse-core/test/audits/deprecations-test.js

```diff
@@ -69,8 +70,8 @@ describe('Console deprecations audit', () => {
     });
     assert.equal(auditResult.rawValue, false);
     assert.equal(auditResult.displayValue, '2 warnings found');
-    assert.equal(auditResult.extendedInfo.value.length, 2);
-    assert.equal(auditResult.extendedInfo.value[0].url, URL);
-    assert.equal(auditResult.extendedInfo.value[0].label, 'line: 123');
+    assert.equal(auditResult.details.items.length, 2);
+    assert.equal(auditResult.details.items[0][1].text, URL);
```

Comment:
you still have to `require` it though and it's meant to be identical so why need to rename?